internal/operands/template-validator/defaults.go

```diff
@@ -1,5 +1,5 @@
 package template_validator
 
-const (
-	defaultTemplateValidatorImage = "quay.io/kubevirt/kubevirt-template-validator:v0.9.0"
+var (
+	DefaultTemplateValidatorImage = "quay.io/kubevirt/kubevirt-template-validator:v0.9.0"
```

Comment:
Does the variable need to be exported? can you change its default value to something obviously incorrect, like `_VALIDATOR_IMAGE_PLACEHOLDER_`.